src/test/java/org/prebid/server/auction/BidResponseCreatorTest.java

```diff
@@ -61,15 +69,6 @@
 import org.prebid.server.settings.model.Account;
 import org.prebid.server.settings.model.VideoStoredDataResult;
 
-import java.io.IOException;
-import java.math.BigDecimal;
-import java.time.Clock;
-import java.time.Instant;
-import java.time.ZoneId;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-
```

Comment:
Intelij idea. Settings. Code Style -> Scheme = Project. If it doesn't help than 
Click to cogwheel -> import code scheme. From checkstyle file. 
Perform code style check formating for all files that you added

Reply:
ALWAYS